src/init.cpp

```diff
@@ -394,6 +394,7 @@ std::string HelpMessage(HelpMessageMode mode)
     }
     strUsage += HelpMessageOpt("-shrinkdebugfile", _("Shrink debug.log file on client startup (default: 1 when no -debug)"));
     strUsage += HelpMessageOpt("-testnet", _("Use the test network"));
+    strUsage += HelpMessageOpt("-requirestandard", _("Require transactions to be standard even in testnet/regtest"));
```

Comment:
Perhaps you should somehow disallow setting it to false on mainnet or only allow setting it on testnet/regtest then? Or extend the help message explaining that fact? Dunno what is best, but an option without a default looks strange IMHO.